matter_server/server/device_controller.py

```diff
@@ -1211,7 +1208,7 @@ async def _on_mdns_operational_node_state(
         try:
             self._mdns_inprogress.add(node_id)
             node = self._nodes[node_id]
-            if state_change == ServiceStateChange.Added:
+            if state_change in (ServiceStateChange.Added, ServiceStateChange.Updated):
```

Comment:
Do we need the same change on line 1229?

Reply:
Let's look at that once we do the follow up PR about commissionable node discovery.